src/ui/prompt.js

```diff
@@ -31,9 +31,12 @@ export class ParamPrompt {
         AssertionError.raise("Unsupported parameter type: " + param.type)
       return this.paramTypes[param.type].render.call(this.pm, param, this.defaultValue(param))
     })
+    let promtTitle = elt("h5", {}, (command.spec && command.spec.label) ? command.spec.label : '')
+    let submitButton = elt("button", {type: "submit", class: "ProseMirror-prompt-submit"}, "Ok")
+    let cancelLink = elt("a", {class: "ProseMirror-prompt-cancel"}, "Cancel")
```

Comment:
That is true, although it's nice to have them both from an UX perspective (the ux team, with we're integrating prosemirror in a usecase, suggested this - along with the prompt title)